stemgen/track.py

```diff
@@ -4,13 +4,11 @@
 from torch import tensor
 import numpy as np
 
-from .constant import SAMPLE_RATE
-
 logger = logging.getLogger(__file__)
 
 
 class Track:
-    def __init__(self, path):
+    def __init__(self, path, sample_rate):
```

Comment:
Could you add the type annotation for this ctor?

```suggestion
    def __init__(self, path: str, sample_rate: int):
```